crates/rome_js_formatter/src/utils/assignment_like.rs

```diff
@@ -115,8 +177,13 @@ impl Format<JsFormatContext> for RightAssignmentLike {
 /// Assignment like are:
 /// - Assignment
 /// - Object property member
+/// - Variable declaration
 #[derive(Debug)]
 pub(crate) enum AssignmentLikeLayout {
+    /// This is a special layout usually used for variable declarations
+    /// This layout is hit, usually, when variable declarator doesn't have initializer:
```

Comment:
I believe you wanted to add an example? The phrase finishes with `:`
```suggestion
    /// This is a special layout usually used for variable declarations.
    /// This layout is hit, usually, when a [variable declarator](JsVariableDeclarator) doesn't have initializer:
```